_posts/randy/2022-09-10-tech-juggling-reality.md

```diff
@@ -0,0 +1,171 @@
+---
```

Comment:
This is an old article right?  Shouldn't be showing up here

Reply:
don't know what

Reply:
there was an overdue typo fix i think